tidy3d/web/webapi.py

```diff
@@ -441,8 +405,9 @@ def delete(task_id: TaskId) -> TaskInfo:
         Object containing information about status, size, credits of task.
     """
 
-    method = f"tidy3d/tasks/{str(task_id)}"
-    return http.delete(method)
+    task = SimulationTask.get(task_id)
```

Comment:
same extra call as abvoe.
Please reduce the get call here for all public functions in webapi.py